GETTING-STARTED.md

```diff
@@ -321,6 +362,46 @@ val setRandomTestId: List[ActionBuilder] = {
 setup("prep", "Prepare for test") withActions (setRandomTestId:_*)
 ```
 
+#### Repeat a request
+A request or requests can be repeated multiple times using Gatling's [repeat](https://gatling.io/docs/gatling/reference/3.4/general/scenario/#repeat) function.
+This is useful when you want to repeat certain requests for a certain number of time. For example, adding 3 items during a journey.
+
+```scala
+def repeatRequests: List[ActionBuilder] = {
+  repeat(3) {
+    exec(addItem)
+      .exec(getItemPage)
+  }.actionBuilders
+}
+
+setup("post-vat-return-period", "Post vat return period")  withRequests(addItem) withActions (repeatRequests:_*)
+```
+
+#### Iterate asLongAs
+Use `asLongAs` to iterate [as long as](https://gatling.io/docs/gatling/reference/3.4/general/scenario/#aslongas) the 
+condition is satisfied and the duration has not been reached.
+
+For example, to repeat a request `asLongAs` the status of a page is **not** `200`:
+
+```scala
+def getTurnoverPage: List[ActionBuilder] = {
+  asLongAs(session => 
+    !session.attributes.get("turnOverPageStatus").contains(200)) {
```

Comment:
Something about this code example looks a little strange, is there a link somewhere to the Gatling API for Scala that I could look at?

Reply:
Hi, @JoPintoPaul. This is where `asLongAs` is defined: https://github.com/gatling/gatling/blob/v3.4.2/gatling-core/src/main/scala/io/gatling/core/structure/Loops.scala#L75